tools/development/helpers/install-python.sh

```diff
@@ -12,7 +12,7 @@ python_directory="${project_directory}/build/bindings/python/OpenSpaceToolkit*Py
 pushd ${python_directory} > /dev/null
 
 python${PYTHON_VERSION} -m pip install plotly pandas
-python${PYTHON_VERSION} -m pip install git+https://github.com/open-space-collective/cesiumpy.git#egg=cesiumpy
+python${PYTHON_VERSION} -m pip install git+https://github.com/open-space-collective/cesiumpy
```

Comment:
Don't you need the `egg=X` to pip install it properly?

Reply:
Don't think it's necessary, I've installed it without the .git#egg=cesiumpy.

Reply:
Then as long as you've tested it locally and it works, then great!